packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -266,11 +265,11 @@ class _BottomSheetState extends State<BottomSheet> {
 
   bool get _dismissUnderway => widget.animationController!.status == AnimationStatus.reverse;
 
-  Set<MaterialState> dragHandleMaterialState = <MaterialState>{};
+  Set<WidgetState> dragHandleState = <WidgetState>{};
```

Comment:
optional nit: Maybe the variable name could reflect how it's a set of WidgetStates.
```suggestion
  Set<WidgetState> dragHandleStates = <WidgetState>{};
```